src/types/types.ts

```diff
@@ -738,7 +738,7 @@ export interface EventChangeById extends EventPatch, InternalId {}
 
 export interface EventFilter extends CreatedAndLastUpdatedTimeFilter {
   startTime?: DateRange;
-  endTime?: DateRange;
+  endTime?: DateRange | { isNull: boolean };
```

Comment:
better to define new type or interface
and then you can add typedoc as well

Reply:
smthing like `NullableDateRange`?

Reply:
`NullableProperty` or `NullableFilterProperty` would be more flexible

Reply:
done